pages/_app.tsx

```diff
@@ -28,18 +29,26 @@ export type NextPageWithLayout = NextPage & {
   getLayout?: (page: ReactElement) => ReactNode;
 };
 
+interface Cookies {
+  themeTracking: string;
+  timestampTracking: string;
+  analyticsTracking: string;
```

Comment:
Using boolean types instead of strings will make the work with those values easier as they only can be `'true'` or `'false'`. You just need to convert them to boolean when you read the cookie values. Changing the name to something like `allowThemeTracking` for example, would be more readable too

Reply:
OK, i will rename , and add boolean type